tools/oss-fuzz.sh

```diff
@@ -56,6 +64,17 @@ fi
 
 ninja -v -C "$build" fuzzers
 
+# Compressed BCD files are kept in test/test-bcd so let's unpack them
+# and populate the directory the script uses to generate seed corpora
+rm -rf test/fuzz/fuzz-bcd/
+mkdir -p test/fuzz/fuzz-bcd/
```

Comment:
You should never touch the source tree like this. The unpacking should be done in a temp folder.

Reply:
I think initially the idea was to keep those files there to make it easier to test it locally. Instead of unzipping that seed corpus into a directory and passing it to the fuzzer I was possible to simply run `./out/fuzz-bcd test/fuzz/fuzz-bcd/` and be done with it. Then again I agree it isn't perfect.

Reply:
The fuzzing infra should really be taught to (optionally) decompress if input if needed you ask me.

Reply:
I'm not sure how libFuzzer can tell compressed seed corpora from zip archives that are supposed to be passed to fuzz targets fuzzing zip files apart though.

Reply:
`mktemp` is not used

Reply:
There was a typo. `mktemp` is now used